### DISCUSSION TITLE
Actor information

### DISCUSSION THREAD

**Discussion by DSyms2022** [cinemagoer/cinemagoer#506] [Q&A]
Hi,
I just started using this great code set, and for movie information it works great. 
I seem to be struggling with the actor information. This is what I have tried:

people = ia.search_person('angelina')
print(people[0])
print (people[0].personID)
person = ia.get_person(people[0].personID)
print(person.infoset2keys)
#print(person['bdate'])
#print(person['birth date'])
print('Bio')
print(person.get('mini biography',''))
print('Birth date')
print(person.get('birth date',''))

and this is what I get:

Angelina Jolie
0001401
{'main': ['headshot', 'imdbID', 'name'], 'filmography': ['filmography'], 'biography': []}
Bio

Birth date

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
